python/lib/dcos/dcos/marathon.py

```diff
@@ -446,6 +446,21 @@ def restart_app(self, app_id, force=False):
         response = self._rpc.http_req(http.post, path, params=params)
         return response.json()
 
+    def delay_reset(self, app_id):
+        """
+        :param app_id: The app_id which is used to delete the delay
```

Comment:
this doc should be updated to refer to pod_id or app_id.  Also... the `app_id` is required for `marathon/main.py` but this id arg name could be changed to reflect what we want.   it might be best to call it `id`